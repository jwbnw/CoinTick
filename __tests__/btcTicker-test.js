import React from 'react';
import BtcTicker from './../components/btcTicker';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<BtcTicker />).toJSON();
  expect(tree).toMatchSnapshot();
});