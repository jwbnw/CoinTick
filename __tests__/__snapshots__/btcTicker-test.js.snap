// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BTC Ticker Render Testing BTC Ticker isLoading: True Renders Correctly 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignSelf": "center",
      "opacity": 1,
      "padding": 10,
    }
  }
>
  <Image
    source={Object {}}
    style={
      Object {
        "alignSelf": "center",
      }
    }
  />
</View>
`;
